packages/core/src/Site/index.js

```diff
@@ -889,6 +919,35 @@ class Site {
     });
   }
 
+  /**
+   * Handles the reloading of ignore attributes
+   */
+  async handleIgnoreReload(oldIgnore) {
+    const assetsToRemoved = _.difference(this.siteConfig.ignore, oldIgnore);
+
+    if (!_.isEqual(oldIgnore, this.siteConfig.ignore)) {
+      await this._removeMultipleAssets(assetsToRemoved);
+      await this.buildAssets();
+    }
+  }
+
+  /**
+   * Handles the reloading of the style attribute if it has been modified
+   */
+  async handleStyleReload(oldStyle) {
+    if (!_.isEqual(oldStyle.bootstrapTheme, this.siteConfig.style.bootstrapTheme)) {
+      if (this.siteConfig.style.bootstrapTheme) {
+        await this.copyBootswatchTheme();
+      } else {
+        const defaultBootstrapTheme = require.resolve('@markbind/core-web/asset/css/bootstrap.min.css');
```

Comment:
It seems like this block of logic is similar to `Site#copyBootswatchTheme()`. Would it be better to modify `copyBootswatchTheme()` to also account for copying the default bootstrap theme and then using calling that function in here instead?

Reply:
Sounds good. I have updated the method to copy the default theme if applicable during live reload of site.json.